packages/lib-classifier/src/hooks/useFreehandLineReductions.js

```diff
@@ -0,0 +1,51 @@
+import { useStores } from '@hooks'
+import { useCaesarReductions } from './'
+
+export default function useFreehandLineReductions() {
+  const {
+    subjectViewer: {
+      frame
+    },
+    workflows: {
+      active: workflow
+    },
+    workflowSteps: {
+      active: step,
+      findTasksByType
+    }
+  } = useStores()
+
+  // NOTE: useFreehandLineReductions() and useTranscriptionReductions() are called regardless of reducer key
+  if (workflow.caesarReducer !== 'machine-learnt') return
+
+  const { loaded, caesarReductions } = useCaesarReductions(workflow.caesarReducer)
+
+  const caesarMark = caesarReductions?.findCurrentTaskMark({
+    stepKey: step.stepKey,
+    tasks: step.tasks,
+    frame,
+  })
+
+  let task = step?.tasks[caesarMark?.taskIndex]
+  let tool = task?.tools[caesarMark?.toolIndex]
+  let marks = tool?.marks;
+  let marksjson = marks?.toJSON()
+
+  if (marks && caesarMark && marksjson[caesarMark.markId] === undefined) {
+    tool.createMark({
+      frame: caesarMark.frame,
+      id: caesarMark.markId,
+      toolIndex: caesarMark.toolIndex,
+      pathX: caesarMark.pathX,
+      pathY: caesarMark.pathY
+    })
+  }
+
+  const [drawingTask] = findTasksByType('drawing')
+  const drawingMarks = drawingTask?.marks
+
+  return {
+    task: drawingTask,
+    marks: drawingMarks,
```

Comment:
This line is overriding the marks that are passed into `DrawingToolMarks`, which leads to weird bugs where marks can be created but aren’t passed to the component and rendered. 

https://github.com/zooniverse/front-end-monorepo/blob/28fc9a5a751131a22a7d8ccc59e942793963dc8b/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js#L182-L193

Reply:
Check it out on this workflow, where there are two drawing tasks but only marks from the first one are shown. The `InteractionLayer` component should pass all drawn marks, from every step of the workflow, into `DrawingToolMarks`.
https://fe-project-branch.preview.zooniverse.org/projects/eatyourgreens/-project-testing-ground/classify/workflow/3370?env=staging

Reply:
This should fix it:
```suggestion
```